addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2197,7 +2197,7 @@ const ListUserValueWidget = UserValueWidget.extend({
             const checkboxes = [...this.listTable.querySelectorAll('we-button.o_we_checkbox_wrapper.active')];
             this.selected = checkboxes.map(el => {
                 const input = el.parentElement.previousSibling.firstChild;
-                const id = input.name || input.value;
+                const id = this.isCustom ? input.value : input.name;
```

Comment:
📢 Will need to be adapted following [this change](https://github.com/odoo/odoo/commit/3dde03308d4adffd861570230edcef1c5f7fc1c4#diff-ffb61e86e6b8297ef8997f9c605de14de896b54161c032d88a80bbaeac4f89beR2098-R2101) in the forward port.